### DISCUSSION TITLE
Enabling Provisioning API in Self Hosted Version

### DISCUSSION THREAD

**Discussion by tcj123** [plausible/analytics#1290] [Self-Hosted Support]
Hi - We have a self-hosted version of Plausible running, is it possible to enable the Provisioning API on the self-hosted version of the application?

**Comment by ukutaht** [comment]
Hey! It will be included in the next release.

**Comment by dinandmentink** [comment]
Found this with a related question. How do I enable an api key for sites provisioning api (https://plausible.io/docs/sites-api) in the self hosted version?

Edit. 

Never mind, found it myself here: https://github.com/plausible/analytics/discussions/1460.
